src/containers/quiz/quiz-info/QuizInfo.tsx

```diff
@@ -158,4 +167,133 @@ const GradedQuizInfo = ({ points, totalPoints }: GradedQuizInfoProps) => {
   )
 }
 
-export { ActiveQuizInfo, FinishedQuizInfo, UngradedQuizInfo, GradedQuizInfo }
+type StartViewQuizInfoProps = {
+  questionsAmount: number
+  attempts: QuizAttempt
+  timeLimit: QuizTimeLimit
+  usedAttempts: FinishedAttempts
+  handleStartButton: (value: boolean) => void
+}
+const StartViewQuizInfo = ({
+  questionsAmount,
+  attempts,
+  timeLimit,
+  usedAttempts,
+  handleStartButton
+}: StartViewQuizInfoProps) => {
+  const { t } = useTranslation()
+  const [isOpen, setIsOpen] = useState(false)
+
+  const limits = {
+    isNoLimitAttempt: attempts === QuizAttempt.NoLimit,
+    isNoLimitTime: timeLimit === QuizTimeLimit.NoLimit,
+    maxAttempts: parseInt(attempts, 10) || 0
+  }
+  const isHaveAttempts =
+    limits.isNoLimitAttempt || usedAttempts.length < limits.maxAttempts
+
+  const typographyStyle = (subType: number) => {
+    return spliceSx(
+      styles[`subtitle${subType}` as keyof typeof styles],
+      styles.subtitleSize
+    )
+  }
+  const getQuizTimeLimitTitle = (t: TFunction, timeLimit: QuizTimeLimit) => {
+    const timeOption = getQuizTimeLimitFields(t).find(
+      (option) => option.value === timeLimit
+    )
+    return timeOption ? timeOption.title : ''
+  }
+
+  const onStartAttempt = () => {
+    limits.isNoLimitTime ? handleStartButton(false) : setIsOpen(true)
+  }
+
+  const attemptLimitOutput = !limits.isNoLimitAttempt && (
+    <>
+      <Box sx={styles.dividerEllipse}>
+        <DividerComponent
+          caption=''
+          orientation='horizontal'
+          size='small'
+          textAlign='center'
+          thickness='md'
+          type='ellipse'
+          variant='middle'
+        />
+      </Box>
+      <Typography sx={typographyStyle(1)}>{t('quiz.attemptLimit')}:</Typography>
+      <Typography sx={typographyStyle(2)}>
+        {usedAttempts.length}/{attempts.split(' ')[0]}
+      </Typography>
+    </>
+  )
+  const timeLimitOutput = !limits.isNoLimitTime && (
+    <>
+      <Box sx={styles.dividerEllipse}>
+        <DividerComponent
+          caption=''
+          orientation='horizontal'
+          size='small'
+          textAlign='right'
+          thickness='md'
+          type='ellipse'
+          variant='inset'
+        />
+      </Box>
+      <Typography sx={typographyStyle(1)}>{t('quiz.timeLimit')}:</Typography>
+      <Typography sx={typographyStyle(2)}>
+        {getQuizTimeLimitTitle(t, timeLimit)}
+      </Typography>
+    </>
+  )
+
+  const noAttemptsAlert = !isHaveAttempts && (
+    <Alert severity='info'>{t('quiz.reachedAttemptLimit')}</Alert>
+  )
+
+  const handleTimeLimitModal = (isOpen: boolean, isStart: boolean) => {
+    setIsOpen(isOpen)
+    handleStartButton(isStart)
+  }
```

Comment:
```suggestion
  }
  
```
Add an indent before `return` statement, please.